distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -93,48 +121,64 @@ Function Update-openHAB {
 
```

Comment:
My fault actually (sorry) but I just noticed: line 117, `mkdir backup` should be `mkdir $OHDirectory\backup`.
The update could be launched from another directory with the `-OHDirectory` parameter explicitely set.